php/php.editor/src/org/netbeans/modules/php/editor/indent/FormatVisitor.java

```diff
@@ -2558,4 +2560,9 @@ protected static boolean isWhitespace(final CharSequence text) {
     private static boolean isAnonymousClass(ASTNode astNode) {
         return astNode instanceof ClassInstanceCreation && ((ClassInstanceCreation) astNode).isAnonymous();
     }
+
+    private static boolean isPHPOperator(PHPTokenId id) {
```

Comment:
Could be maybe moved to some `util` class as it seems to me to be quite useful in general. Not sure about that so just a hint.

Reply:
Will do it. Thanks!